test/integration/validation.spec.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  ValidationDataClient,
+  Product,
+  ConstantProduct,
+  ChildProduct
+} from "./generated/validation/src";
+import { assert } from "chai";
+import * as fs from "fs";
+
+const constantBody: Product = {
+  child: { constProperty: "constant" },
+  constChild: {
+    constProperty: "constant",
+    constProperty2: "constant2"
+  },
+  constInt: "undefined",
+  constString: "constant"
+};
+
+describe("Integration tests for validation", () => {
+  let client = new ValidationDataClient("");
+
+  it("constant in path", async () => {
+    client.getWithConstantInPath();

```

Comment:
I think you need to `await` your client calls

Reply:
Yup, I just released I was not looking at the results of the tests and I assumed they were passing. Let me push a fix.